curvefs/src/client/block_device_client.h

```diff
@@ -25,80 +25,148 @@
 #define CURVEFS_SRC_CLIENT_BLOCK_DEVICE_CLIENT_H_
 
 #include <memory>
+#include <utility>
+#include <string>
 
 #include "src/client/client_common.h"
 #include "src/client/libcurve_file.h"
 #include "curvefs/src/client/config.h"
 #include "curvefs/src/client/error_code.h"
 
-using ::curve::client::FileClient;
-
 namespace curvefs {
 namespace client {
 
+#define CURVE_ALIGN(d, a) (((d) + (a - 1)) & ~(a - 1))
```

Comment:
change this to a function, and it probably is inlined.

Reply:
> change this to a function, and it probably is inlined.

yep, i will improve it.